tutorial/getting-started/README.md

```diff
@@ -0,0 +1,10 @@
+# Getting started
+
+A Javascript getting started tutorial to `bpmn-visualization` using an online monitoring scenario.
+- [__⏩ live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/tutorial/getting-started/index.html)
+- to run locally, open the [index.html](index.html) directly in a Web Browser
+
+## ♻️ Usage
+To see the content of the iframe: https://process-analytics.github.io/bpmn-visualization-tutorial-getting-started.
+
+Its code is on the [solution](https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started/tree/solution) branch of [bpmn-visualization-tutorial-getting-started](https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started) repository. 
```

Comment:
**Proposal**

The code of the tutorial is hosted in a dedicated repository: https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started/

This example displays the solution of the tutorial in an iframe. The solution itself is hosted at https://process-analytics.github.io/bpmn-visualization-tutorial-getting-started.

Reply:
LGTM 🙂